frontend/webservice/node.cgi

```diff
@@ -148,10 +160,19 @@ sub create_node {
             controller => $params->{controller}{value}
         }
     );
-
-    my $create_err = $node->create;
+    
+    my $create_err;
+    eval{
+        $create_err = $node->create;
+    };
```

Comment:
Don't worry about using eval around these db methods. From what I understand, by default exceptions are not raised by perl DBI. Instead the results of the methods are used to determine success / failure.